hooks/environment

```diff
@@ -12,6 +12,22 @@ check_command() {
   fi
 }
 
+# the buildkite agent redactor has been added from version 3.66.0 and on. So we need to check it.
+# https://github.com/buildkite/agent/releases/tag/v3.66.0
```

Comment:
nit: is this link needed?

Reply:
It is needed to verify the comment

Reply:
OK, then in which case, let's tweak the comment (see below)